server/schemas/typeDefs.js

```diff
@@ -29,13 +29,19 @@ const typeDefs = gql`
   }
 
   type Room {
-    _id: ID
-    roomType: String
-    bedType: String
-    roomView: String
+    id: ID
+    roomType: String!
     price: Int
+    roomCount: Int
   }
 
+  input RoomInput {
```

Comment:
looks like lines 38 to 43 are the same as 31 through 36

Reply:
Yes, this is not needed due to the work around. GraphQL would need one for Input, and one for Output.
When using [Room] as a type to send in the reservation, it would not work because [Room] is output only.
Thank you for pointing it out, will clean it up.